test/recipe/admin/utils/giveMe.js

```diff
@@ -52,6 +52,25 @@ export default {
     return { key: keyAndTestId, title, component };
   },
 
+  anErrorPage({ safeLinks, errorResults, data}){
+    const errorInfos = {};
+    errorResults.forEach(element => {
+      errorInfos[element.statusCode] = { ...element };
+    });
+
+    return {
+      type: "ErrorPage",
+      schema: {
+        safeLinks,
+        errorInfos
+      },
+      data: {
+        type: "Inline",
+        value: data
+      }
```

Comment:
make these single line, occupies too much space. (other stubbers are reformatted in #226 )